### PR TITLE
Add Github Link to repo on Grant Detail info

### LINKED ISSUE #7270: grants - - there should be a github link to their repo on the grantdetail page
there should be a github link to their repo on the grantdetail page
and on grants/new (an optional field)

### PR DESCRIPTION
**Pull request by molecula451**
##### Description

This PR will optionally request github link repo on the grants new form and append it to the grant details info for the looks

Screencast-2020-08-30-124937

##### Refers/Fixes

fix #7270

### CODE REVIEW

File: app/grants/templates/grants/detail/info.html [hunk 1]
```diff
@@ -101,6 +101,18 @@ <h1 class="font-title-xl my-4 font-weight-bold">
           </p>
         </div>
       {% endif %}
+        
+        {% if grant.github_project_url %}
```
Comment (thelostone-mc) [inline app/grants/templates/grants/detail/info.html:105]: deindent this by 2

File: app/grants/templates/grants/detail/info.html [hunk 2]
```diff
@@ -101,6 +101,18 @@ <h1 class="font-title-xl my-4 font-weight-bold">
...
+        <div class="font-body mb-2">
+          <i style="width: 14px;" class="fab fa-github-square mr-2 text-center"></i>
+          <p class="d-inline-block mb-0">
+            <a href="{{ grant.github_project_url }}" target="_blank" rel="" data-toggle="tooltip" data-html="true" data-placement="top" for="github_project_url" 
+            title="Project Github URL">
+              {{ grant.github_project_url }}
+            </a>
+          </p>
+        </div>
```
Comment (thelostone-mc) [inline app/grants/templates/grants/detail/info.html:114]: indent this by 1

**Comment by thelostone-mc** [comment]
@molecula451 looks good except for the indentation!Could you fix that up ?

File: app/grants/templates/grants/detail/info.html (hunk 1 above)
Comment (molecula451) [reply app/grants/templates/grants/detail/info.html]: de-indented.

File: app/grants/templates/grants/detail/info.html (hunk 2 above)
Comment (molecula451) [reply app/grants/templates/grants/detail/info.html]: indented.

**Comment by molecula451** [comment]
> @molecula451 looks good except for the indentation!Could you fix that up ?

yup. fixed :)

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**